src/pages/index.js

```diff
@@ -8,11 +9,13 @@ import SectionContact from '../components/SectionContact'
 import SectionBenefits from '../components/SectionBenefits'
 import SectionHome from '../components/SectionHome'
 import FloatingButtons from '../components/FloatingButtons'
+import BlogSection from '../components/BlogSection'
 import SectionFaq from '../components/SectionFaq'
 
-export default function Index({location}) {
+export default function Index({data, location}) {
     const url = location.href ? location.href : '';
-
+    const allPosts = data.allWpPost.nodes
+    const fewPosts = [...allPosts.slice(-3)]
```

Comment:
The _slice_ function already returns an array. So you don't need to spread it inside an array again. Just `const fewPosts = allPosts.slice(-3)` should do the job. However, if we are using just three posts here, it's better to put this limit in the query. This way we avoid querying unnecessary data. The sorting can also be done in the query, as shown in the comment below.